utils/askForRestart.js

```diff
@@ -0,0 +1,21 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { GAME_MESSAGES } from "../constants/messages.js";
+
+/**
+ * 사용자에게 재시작 or 종료를 묻는 함수
+ * @returns boolean
+ */
+const askForRestart = async () => {
+  const restart = await MissionUtils.Console.readLineAsync(
+    GAME_MESSAGES.RESTART
+  );
+
+  if (restart === "1") {
+    return true;
+  }
+  if (restart === "2") {
+    return false;
+  }
```

Comment:
```suggestion
  if (restart === "1") {
    return true;
  }
  if (restart === "2") {
    return false;
  }
  
  throw new Error("에러 처리");
```
유저 입력이 1, 2가 아닌 경우에 에러 처리가 필요할 것 같아요~!